src/script/acquisition-sdk.ts

```diff
@@ -104,6 +115,7 @@ export class AcquisitionManager {
 
             if (response.statusCode !== 200) {
                 let errorMessage: any;
+                this.disableApiCalls(response.statusCode)
```

Comment:
naming is incorrect. The function doesn't necessarily disables calls, it rather checks status code and decide if need to disable. Should be something like "HandleRequestFailure".